codegen/src/test/resources/software/amazon/awssdk/codegen/emitters/customizations/processors/uselegacyeventgenerationscheme/service-2.json

```diff
@@ -452,14 +452,23 @@
       "type": "structure",
       "members": {
         "InputEventOne": {
-          "shape": "InputEvent"
+          "shape": "InputEventOne"
         },
         "InputEventTwo": {
           "shape": "InputEventTwo"
         }
       },
       "eventstream": true
     },
+    "InputEventOne": {
```

Comment:
There are a number of existing test models which have shared events like this - They don't cause issues in the current test because they aren't used to generate/build a full SDK.

We have a few options, we can:
1.  Modify the test model to duplicate the shape (as is done here).
2. Add the new customization flag to customization.config for each model to do the rename.
3.  Disable the new processor for these cases